llvm-general/Setup.hs

```diff
@@ -124,7 +132,9 @@ main = do
               libraryCondTree <- condLibrary genericPackageDescription
               return libraryCondTree {
                 condTreeData = condTreeData libraryCondTree <> mempty {
-                    libBuildInfo = mempty { ccOptions = llvmCxxFlags }
+                    libBuildInfo =
+                      mempty { ccOptions = llvmCxxFlags
+                             , extraLibs = [stdLib] }
```

Comment:
Please break lists according to the style used elsewhere in llvm-general: commas at the end of the line, minimize dependence of indentation on the content of the prior line.

I'd break before ccOptions and use a fixed indent rather than indent extraLibs a distance dependent on the length of "mempty". (This choice aims to minimize time spent adjusting whitespace to keep things aligned. Yes it sometimes conflicts with Haskell's whitespace dependencies).

All such style choices are debatable, but consistency first.